keyboards/yanghu/unicorne/keymaps/default/rules.mk

```
@@ -0,0 +1,3 @@
+ifeq ($(strip $(OLED_DRIVER_ENABLE)), yes)
+  SRC += oled.c
+endif
```

This may not work well, would be simpler to always include it, and wrap the file in ifdefs.
```suggestion
SRC += oled.c
```